lib/dradis/plugins/saint/importer.rb

```diff
@@ -0,0 +1,97 @@
+module Dradis::Plugins::Saint
+  class Importer < Dradis::Plugins::Upload::Importer
+    def import(params={})
+      @issues = {}
+      @hosts = {}
+      file_content = File.read(params[:file])
+
+      logger.info {'Parsing SAINT output file...'}
+      doc = Nokogiri::XML( file_content )
+      logger.info{'Done.'}
```

Comment:
Can you add an intermediate step here before parsing the document that runs a check on the document root's to ensure that the file is actually in the expected format? See dradis-nessus or dradis-appscan.

Reply:
✅